README.md

```diff
@@ -1,3 +1,8 @@
+> **🚀 You are looking at a pre-prelease of tRPC v10!**
```

Comment:
Is `pre-prelease` intentional wording? I know this was copied from the header but we should probably fix it.

Reply:
It's okay